Documentation/gettingstarted/encryption.rst

```diff
@@ -85,6 +85,29 @@ to GKE, with VXLAN tunneling:
       --set encryption.enabled=true \\
       --set encryption.nodeEncryption=false
 
+On GKE, Cilium can also be deployed in Direct Routing mode, without tunneling.
+This requires us to enable the GKE integration and specify the native routing
+CIDR.
```

Comment:
The text can be read to imply that the difference with previous example is just that we use routing, but in the command below we set `encryption.nodeEncryption=true` which I think is confusing. I would suggest we either mention in the paragraph here that we also enable `nodeEncryption`, or just use `encryption.nodeEncryption=false`

Reply:
Agreed, clarified as: https://github.com/cilium/cilium/commit/e8c3490e7f44ea468a5e068d1bf080bca83f70f0#diff-5c5b074c95a92e31819b37002f43e2763a495a4155886e3cfcf5b7848b3e8605R90-R91